src/containers/access/src/EditAccessRequestContainer.js

```diff
@@ -1,6 +1,12 @@
 // @flow
+import { useRef } from 'react';
 
+import styled from 'styled-components';
+import { faPrint } from '@fortawesome/pro-regular-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { IconButton } from 'lattice-ui-kit';
 import { DataUtils, ReduxUtils } from 'lattice-utils';
+import { useReactToPrint } from 'react-to-print';
```

Comment:
No change required, just a thought. Maybe the parent app should be responsible for this dependency, and all the module does is pass along the ref in a `onPrint` callback or something.